.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "remote"]
 	path = remote
-	url = git@github.com:MusicConnectionMachine/api.git
+	url = https://github.com/MusicConnectionMachine/api.git
```

Comment:
ok, then we'll fix this when we actually set up Travis.
@kordianbruck what is the reason that we shouldn't do it this way? (and shouldn't it work with ssh if you define a deploy key?)

BTW: @MusicConnectionMachine/group-4 and @MusicConnectionMachine/group-2 you have solved it the same way!